compiler/src/iree/compiler/Dialect/Modules/VMVX/IR/VMVXOps.td

```diff
@@ -18,4 +18,131 @@ class VMVX_PureOp<string mnemonic, list<Trait> traits = []> :
 // VMVX Ops: ABI
 //===----------------------------------------------------------------------===//
 
+def VMVX_AddOp : VMVX_Op<"add", [SameVariadicOperandSize]> {
+  let summary = "Performs a strided elementwise add of two same-rank buffers";
+  let description = [{
+    Performs addition in-place as if:
+      OUT = LHS + RHS
+
+    All operands have the same rank.
+  }];
+  let arguments = (ins
+    // LHS.
+    VMVX_Buffer:$lhs_buffer,
+    VMVX_Index:$lhs_offset,
+    Variadic<VMVX_Index>:$lhs_strides,
+    // RHS.
+    VMVX_Buffer:$rhs_buffer,
+    VMVX_Index:$rhs_offset,
+    Variadic<VMVX_Index>:$rhs_strides,
+    // OUT.
+    VMVX_Buffer:$out_buffer,
+    VMVX_Index:$out_offset,
+    Variadic<VMVX_Index>:$out_strides,
+
+    // Dimensions.
+    Variadic<VMVX_Index>:$size
+  );
+
+  let assemblyFormat = [{
+    `lhs` `` `(` $lhs_buffer `offset` $lhs_offset `strides` `[` $lhs_strides `]` `:` type($lhs_buffer) `)`
+    `rhs` `` `(` $rhs_buffer `offset` $rhs_offset `strides` `[` $rhs_strides `]` `:` type($rhs_buffer) `)`
+    `out` `` `(` $out_buffer `offset` $out_offset `strides` `[` $out_strides `]` `:` type($out_buffer) `)`
+    `size` `` `(` $size `)`
+    attr-dict
+  }];
+}
+
+def VMVX_CopyOp : VMVX_Op<"copy", [SameVariadicOperandSize]> {
+  let summary = "Copy from one buffer to another";
+  let arguments = (ins
+    // LHS.
+    VMVX_Buffer:$inp_buffer,
+    VMVX_Index:$inp_offset,
+    Variadic<VMVX_Index>:$inp_strides,
+    // OUT.
+    VMVX_Buffer:$out_buffer,
+    VMVX_Index:$out_offset,
+    Variadic<VMVX_Index>:$out_strides,
+
+    // Dimensions.
+    Variadic<VMVX_Index>:$size
+  );
+  let assemblyFormat = [{
+    `inp` `` `(` $inp_buffer `offset` $inp_offset `strides` `[` $inp_strides `]` `:` type($inp_buffer) `)`
```

Comment:
`input` or `in` or `ins` instead (and a matching `output` `out` `outs`)? `inp` is unique to this and not worth the 2 character savings - the fewer abbreviations the better for readability

Reply:
Went with in/out.